src/app.js

```diff
@@ -1 +1,24 @@
 'use strict';
+
+const terminal = require('./modules/terminal');
+const fs = require('fs');
+
+function copyFile() {
+  terminal.question('Enter your command: ', (key) => {
+    const [command, firstFile, secondFile] = key.split(' ');
+
+    if (command === 'cp' && !fs.existsSync(secondFile)) {
```

Comment:
You should also check if the source file exists before trying to copy it. If it doesn't exist, the program will throw an error.